README.md

```diff
@@ -104,7 +104,7 @@ This means that you control how many operations can be executed concurrently.
 If you add a job to the queue and it still below the limit, it will be executed
 immediately. If you keep adding new jobs to the queue and its concurrency limit
 is reached, it will not start a new operation and instead queue this for future
-execution. Once one of the pending operations complete, it will pick the next
+execution. Once one of the pending operations is complete, it will pick the next
```

Comment:
Note that the exact same sentence can also be found in the `Queue` class, I would like to make sure this is consistent. In either case I'm not sure about this one, feels like both would be valid variants, no?

Reply:
Would agree to this. Didn't saw it in the `Queue` class. I was fast eyeballing the sentences, so next time I will take more time in future PR's. Thank you. 👍